### ISSUE TITLE
Error: failed to connect: failed to change firewall state : failed to execute shell command: exit status 3

### ISSUE THREAD

**Issue by JHWelch** [ivpn/desktop-app-cli#7]
# Bug report

## Describe your environment

* Device: HP Desktop 
* OS name and version: Ubuntu 18.04.4 LTS
* IVPN app version: 2.12.4 (date:2020-06-30 commit:9bb26c28c067aba702c1a3e32ea1d1716d179fdd)  

## Describe the problem

This looks to be the same bug that gaalcaras had [here in Issue 5](https://github.com/ivpn/desktop-app-cli/issues/5). I am able to connect if I add the flag `-fw_off` however cannot connect without it.  

### Steps to reproduce:

Try to connect to any server without specific flags. Tried a variety of them. 

### Observed Results:

Unlike gaalcaras, I was able to get a log file.
```
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.679 [prtcl ] Client connected: 127.0.0.1:48806
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.679 [prtcl ] [<--] Hello
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.679 [prtcl ] Connected client version: '1.0' [set KeepDaemonAlone = true]
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.679 [prtcl ] [-->] HelloResp
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.680 [prtcl ] [<--] GetServers
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.680 [prtcl ] [-->] ServerListResp
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [prtcl ] [<--] SetPreference
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [servc ] preferences enable_obfsproxy='false'
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [prtcl ] [<--] Connect
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [prtcl ] Enter VPN status checker
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [servc ] Connecting...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [servc ] Initializing...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [prtcl ] [-->] VpnStateResp
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [servc ] Route change receiver started
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.682 [servc ] VPN state forwarder started
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.686 [ovpn  ] OpenVPN version:[2 4 4]
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.686 [servc ] Initializing firewall
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.686 [frwl  ] Getting status...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.689 [frwl  ] #011false
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.689 [frwl  ] Enabling...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.695 [frwl  ] ERROR firewall.go:64:(in github.com/ivpn/desktop-app-daemon/service/firewall.SetEnabled): failed to execute shell command: exit status 3
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.695 [servc ] ERROR service.go:625:(in github.com/ivpn/desktop-app-daemon/service.(*Service).connect): Failed to enable firewall:failed to change firewall state : failed to execute shell command: exit status 3
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.695 [frwl  ] Disabling...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.737 [frwl  ] Getting status...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [frwl  ] #011false
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [prtcl ] [-->] KillSwitchStatusResp
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [servc ] Route change receiver stopped
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [servc ] VPN state forwarder stopped
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [servc ] VPN process stopped
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [servc ] ERROR service.go:390:(in github.com/ivpn/desktop-app-daemon/service.(*Service).keepConnection): Connection error: failed to change firewall state : failed to execute shell command: exit status 3
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [prtcl ] ERROR protocol.go:834:(in github.com/ivpn/desktop-app-daemon/protocol.(*Protocol).processRequest): failed to change firewall state : failed to execute shell command: exit status 3
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [prtcl ] [-->] DisconnectedResp
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [servc ] Requesting session status...
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [prtcl ] Exit VPN status checker
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [prtcl ] [<--] Disconnect
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.741 [prtcl ] [-->] DisconnectedResp
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.742 [prtcl ] Client disconnected: 127.0.0.1:48806
Jul 31 17:23:07 Media-Jordan ivpn-service[844]: Jul 31 17:23:07.742 [prtcl ] Current state not changing [KeepDaemonAlone=true]
Jul 31 17:23:08 Media-Jordan ivpn-service[844]: Jul 31 17:23:08.013 [servc ] Session status request: done
```

### Expected Results:

Connect to a server.

**Comment by stenya** [comment]
Hi @JHWelch,

can you, please, check the output of the command `sudo iptables -L` in a command-line terminal?

**Comment by JHWelch** [comment]
Sure! Here is the output
```
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination
```

**Comment by stenya** [comment]
Thank you. I will try to analyze the problem. 
Probably, I will come to you with more questions later )

**Comment by stenya** [comment]
The issue occurs for users who have Linux OS with disabled IPv6 protocol.
We will release a fix soon.

**Comment by stenya** [comment]
Sorry for the delay.
The issue is fixed in the daemon v2.12.8 
Precompiled versions are available here: https://www.ivpn.net/apps-linux
